_posts/2020-09-18-ai-ethics-optimization-problem/ai-ethics-optimization-problem.Rmd

```diff
@@ -0,0 +1,172 @@
+
+Things get further out of control at the other side of the timeline. Here we have historical bias:
+
+> Historical bias arises when there is a misalignment between the world as it is and the values or objectives to be encoded and propagated in a model. It is a normative concern with the state of the world, and exists even given perfect sampling and feature selection.
+
+I'll get back to why this is called "historical bias" later; the definition is a lot broader though. Now definitely we're beyond the realm of formalization: We're entering the realm of normativity, of ways of viewing the world, of ethics.
+
+Please don't get me wrong. I'm not criticizing the paper; in fact, it provides a useful categorization that may be used as a "check list" by practitioners. But given the formalist style, a reader is likely to focus on the readily-formalizable parts; it is then easy to dismiss the two others as not really being relevant to one's work, and certainly not something one could exert influence on. (I'll get back to that later.)
+
+One little aside before we leave formalism: I do believe that a lot of the formalist work on *AI* ethics is done in good intent; however, work in this area also benefits organizations that undertake it. Citing [Tom Slee](https://tomslee.github.io/publication/oup_private_sector_ai/),
+
+> Standards set public benchmarks and provide protection from future accusations. Auditable criteria incorporated into product development and release processes can confirm compliance. There are also financial incentives to adopt a technical approach: standards that demand expertise and investment create barriers to entry by smaller firms, just as risk management regulations create barriers to entry in the financial and healthcare industries.
+
+## Not everything in life is an optimization problem
+
+Stephen Boyd, who teaches convex optimization at Stanford, is said to often start the introductory lecture with the phrase "everything is an optimization problem". [^5] It sounds intriguing at first; certainly a lot of things in my life can be thought of like that. It may become awkward once you start to compare, say, time spent with loved ones and time spent working; it becomes completely unfeasible when comparing *across people*.
+
+[^5]: At least he does so here: <https://www.youtube.com/watch?v=McLq1hEq3UY>
+
+We saw that even under formalist thinking, there is no impartial way to optimize for fairness. But it's not just about choosing between different types of fairness. How do you weigh fairness against stakeholder interests? No algorithm will be deployed that does not serve an organization's purpose.
```

Comment:
I know this post is a personal opinion, so no need to change... but personally I would say:

> there is **currently** no impartial way to optimize for fairness.

There is none today, but would want to think that one day we figure out a way to optimize fairness, even if it takes solving "General AI" and creating a digital overload to tell us how to behave properly.

Reply:
Hi Javier, thanks for commenting. I know we agree to disagree here :-)

To me fairness is intrinsically a social construct, and so, per definitionem out of reach for any formal approach.